dsc/workflows/__init__.py

```diff
@@ -3,8 +3,8 @@
 All primary functions used by CLI are importable from here.
 """
 
-from dsc.workflows.base import BaseWorkflow
+from dsc.workflows.base import Workflow
+from dsc.workflows.base.simple_csv import SimpleCSV
+from dsc.workflows.demo import DemoWorkflow
 
-__all__ = [
-    "BaseWorkflow",
-]
+__all__ = ["DemoWorkflow", "SimpleCSV", "Workflow"]
```

Comment:
I didn't comment on this earlier, but I continue to really like this.  Not only does it import and therefore register all workflows as a `__subclass__` of `Workflow`, but it allows for this:

```python
from dsc.workflows import DemoWorkflow
```